src/main/java/com/atm/controller/AtmController.java

```diff
@@ -133,11 +135,24 @@ public String history(ModelMap model) {
         return "history";
     }
 
+    @GetMapping({"/top10history"})
+    public String top10History(ModelMap model) {
+        model.addAttribute("transaction_list",transactionService.getTransactionList(loginAccount.getAccountNo()));
+        model.addAttribute("acct", loginAccount.getAccountNo());
+        model.addAttribute("acctName", loginAccount.getName());
+        model.addAttribute("balance", accountService.getAccountDetail(loginAccount.getAccountNo()).getBalance());
+        return "history_list";
+    }
+
     @PostMapping({"/history"})
     public String getHistory(TrxSearchParam param, ModelMap model){
-        String startDate = Constant.dft.format(Constant.dft_notime.parse(param.getStartDate()+" 00:00:00"));
-        String endDate = LocalDate.parse(Constant.dft.format(Constant.dft_notime.parse(param.getEndDate()+" 00:00:00")),Constant.dft).plusDays(1).toString();
-        model.addAttribute("transaction_list",transactionService.getByDateRange(startDate, endDate));
+        System.out.println("thoni "+param.getStartDate());
+        String[] startdate = param.getStartDate().split("/");
+        String[] endate = param.getEndDate().split("/");
+        LocalDateTime start = LocalDateTime.of(Integer.valueOf(startdate[2]),Integer.valueOf(startdate[0]),Integer.valueOf(startdate[1]),00,00,00);
```

Comment:
@akhmad-fathoni Instead of doing the splits, you could parse using `LocalDateTime.parse("your string")`. Also, you might consider setting the beginning and end of the day, if you think it applies.